---
title: Resume
layout: default
navigation_weight: 2
icon: icon fa fa-edit
permalink: /resume/
---

<div id="research" class="page">
	<div class="pageheader">

		<div class="headercontent">

			<div class="section-container">
				<h2 class="title">Research Summary</h2>
			
				<div class="row">
					<div class="col-md-8">
					<p>I have been working on analyzing RNA-seq data of soybean developing embryos for the past 3 years. I have developed a computational pipeline for analysis of this RNA-seq data that has a potential to be applied to any species with known reference genome. I have developed and used several algorithms and tools for mining of this time-course RNA-seq data. In addition to finding differentially expressed genes, I have analyzed differentially expressed splice variants of known genes leading to investigation of alternative splicing events such as exon skipping, intron retention, and antisense transcripts. We have further analyzed these splice variants regarding to their domain composition, coding potential, signaling and metabolic pathways, and co-expression network. </p> 
					<p>Particularly, I am interested in long noncoding RNAs (lncRNAs) and long intergenic RNAs (lincRNAs). lncRNAs and lincRNAs are regulatory RNAs that  their functions have been implicated in many diseases such as cancer. For this purpose, I have developed an SVM classifier with over 98% accuracy that can classify transcripts into coding and noncoding classes with integration of several features related to sequence and structure of a transcript. This SVM classifier is currently trained, tested, and applied on Arabidopsis and soybean transcripts. I am going to extend this classifier to human, mouse, zebrafish, and Drosophila. </p>
				   
					<!-- <div class="col-md-10 col-md-offset-1"> -->
					<ul class="ul-withdetails">
					
					<li>
								<div class="row">
									
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<p style="color:#04B45F">Clink here for exploring computational pipeline that I have developed for analysis and mining of a RNA-seq data.</p>
										</div>
									</div>
								</div>
								<div class="details">
								 <img alt="image" src="img/lab/Soy3_Pipeline_small.png" class="img-responsive">
								
								</div>
							</li>
					
					
					</ul>
			  <!--      </div> -->
				   
					</div>
					<div class="col-md-4">
						<div class="subtitle text-center">
							<h3>Interests</h3>
						</div>
						<ul class="ul-boxed list-unstyled">
							<li>Big Data Analysis</li>
							<li>Bioinformatics, Computational Biology, and Computational Genomics</li>
							<li>Machine Learning and Deep Learning</li>
							<li>Gene Regulatory Networks</li>
							<li>Next Generation Sequencing</li>
							<li>Long noncoding RNAs</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
	</div>

						   
		<div class="section color-2">
			<div class="section-container">
				
				<div class="row">
				 <!--   <div class="col-md-10 col-md-offset-1"> -->
						<div class="title text-center">
							<h3>Selected Research Projects</h3>
						</div>
						<ul class="ul-withdetails">
							<li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/soybeans.png" class="img-responsive">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>Functional Characterization of Expressed Transcripts During Soybean Embryo Development</h3>
											 <p style="color: #04B45F">Click here for a short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
								<p>We have detected all transcripts that were expressed during soybean embryo development. These transcripts include known transcripts, intergenic transcripts, and novel splice variants of known genes. We have investigated all these transcripts for their coding potential, and their domains. </p>
								<p>We have detected 10,798 genes that changed significantly at least in one of the time-points. We have studied these genes using gene ontology enrichment analysis and MapMan metabolic pathway analysis. </p>
								
								</div>
							</li>
							
							<li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/AS.png" class="img-responsive" height="400" width="400">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>Changes in RNA Splicing in Developing Soybean (Glycine max) Embryos</h3>
											 <p style="color: #04B45F">Click here for a short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
								<p>Developing soybean seeds accumulate oils, proteins, and carbohydrates that are used as oxidizable substrates providing metabolic precursors and energy during seed germination. The accumulation of these storage compounds in developing seeds is highly regulated at multiple levels, including at transcriptional and post-transcriptional regulation. RNA sequencing was used to provide comprehensive information about transcriptional and post-transcriptional events that take place in developing soybean embryos. Bioinformatics analyses lead to the identification of different classes of alternatively spliced isoforms and corresponding changes in their levels on a global scale during soybean embryo development. Alternative splicing was associated with transcripts involved in various metabolic and developmental processes, including central carbon and nitrogen metabolism, induction of maturation and dormancy, and splicing itself. Detailed examination of selected RNA isoforms revealed alterations in individual domains that could result in changes in subcellular localization of the resulting proteins, protein-protein and enzyme-substrate interactions, and regulation of protein activities. Different isoforms may play an important role in regulating developmental and metabolic processes occurring at different stages in developing oilseed embryos. </p>    
								</div>
							</li>
							
							
							<li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/SVM.png"  class="img-responsive">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>An accurate support vector machine classifier for assessing coding potential of transcripts using several sequential and structural features</h3>
											 <p style="color: #04B45F">Click here for a short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
								<p>Emergence of next generation sequencing has revealed significant levels of transcriptional activity within both unannotated and annotated regions of the genome, leading to construction of novel tramscripts. These novel transcripts may be located in the genic regions such as antisense, overlapped intronic, and overlapped exonic or may be located in the intergenic regions. However, they can be coding or noncoding in the broader aspect. Hence, one of the main tasks is to functionally characterize these novel transcripts and to determine if they are coding or noncoding (ncRNA). Although the functions of coding genes have been studied for many years, trends to characterize the function of noncoding transcripts have been started recently. Several evidences for implication of ncRNAs in control of development, growth, and disease have been reported so far. ncRNAs can perform their functions through different mechanisms such as chromatin modifications (epigenetic control of gene expression), RNA-protein interactions, and transcriptional inference. </p>
								<p>We propose a support vector machine classifier, which can classify novel transcripts into coding or noncoding with over 96% accuracy. 
								Several sequential and structural features have been compiled for training the classifier. The classifier has been used to classify the novel assembled transcripts from RNA-sequencing pipelines for Soybean and Arabidopsis organisms. 
								However, it can be adapted to any species. </p>
								
								</div>
							</li>
							<li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/Expresso.png"  class="img-responsive">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>Expresso: Exploring Arabidopsisâ€™ transcription factors and target genes from ChIP-Seq data</h3>
											<p style="color: #04B45F">Click here for a short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
								<p>The recent ample availability of Chromatin Immunoprecipitation Sequencing (ChIP-Seq) data presents a distinct opportunity to study the action of transcription factors, particularly in plant science. Even though in vivo transcriptional regulation often involves the concerted action of more than one transcription factor, the format of each individual ChIP-Seq dataset usually represents the action of only one transcription factor. Therefore, a relational database in which all available ChIP-Seq datasets are curated is essential and necessary.
								</p>
								<p>We present Expresso (database and webserver) as a tool for collection and integration of all available Arabidopsis ChIP-Seq data, which in turn can be linked to a userâ€™s gene expression data. Known target genes of transcription factors were identified by motif analysis on publicly available ChIP-Seq data in GEO DataSets. Expresso currently provides three services for identifying: 
								1) the target genes of a given transcription factor, 
								2) the transcription factors that regulate a gene of interest, 3) the correlation of gene expression between transcription factors and their target genes. </p>
								<p> Expresso is currently under development. </p><a href="http://bioinformatics.cs.vt.edu/~expresso/">Check out Expresso here!</a>
								
								</div>
							</li>
							<li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/BN2.png"  class="img-responsive" height="400" width="400">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>Modeling and Identifying Regulatory Modules in (Glycine max) Soybean Time Series Gene Expression Data using Bayesian Networks</h3>
											<p style="color: #04B45F">Click here for a short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
								<p> 
								
								Much of the cellular activity is organized as a network of interacting modules, which are the set of genes interacting together in different conditions to make the cell responsive to different stresses and conditions in the environment as well as inside the cell. 
							Recently, there has been much interest in reverse engineering these networks using gene expression data. One of the popular methods for reverse engineering of these regulatory networks using gene expression data is using Bayesian networks. </p> 
							<p>Gene expression data by itself suffers from high noise and lack of enough power for inferring causal relationship between genes.
							Incorporating prior biological knowledge can improve performance and can help to produce more biologically meaningful network. 
Current project: The goal is to learn a Bayesian network for selected soybean genes from time series gene expression data of soybean developing embryos produced by high throughput sequencing technique (RNA-seq). We take advantage of multiple data types, including co-expression and co-localization for interaction between Arabidopsis homologs of soybean genes as prior knowledge by incorporating these data into gene expression data for learning structure of Bayesian network using Markov Chain Monte Carlo (MCMC) structure learning algorithm. 

								
								</p>
								
								</div>
							</li>
							
							<!-- <li>
								<div class="row">
									<div class="col-sm-4 col-md-3">
										<div class="image">
											<img alt="image" src="img/lab/400x400.png"  class="img-responsive">
											<div class="imageoverlay">
												<i class="icon icon-search"></i>
											</div>

										</div>
									</div>
									<div class="col-sm-8 col-md-9">
										<div class="meta">
											<h3>Title of Preject</h3>
											<p>Very short description of the project.</p>
										</div>
									</div>
								</div>
								<div class="details">
									<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
									<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
									<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
								</div>
							</li> -->
							
						</ul>
					</div>
				</div>
		  <!--  </div> -->
		</div>    
	</div>
